bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/discovery/SensorDiscoveryService.java

```diff
@@ -133,9 +133,16 @@ public void handleReading(Reading reading) {
             logger.debug("discovery for bridge {} found already known sensor id {}", bridge.getThing().getUID(), id);
         }
     }
+    
+    
 
     @Override
     public Class<Reading> getReadingClass() {
         return Reading.class;
     }
+
+    @Override
+    public String getSensorType() {
+        return "ALL";
```

Comment:
Wouldn't it make sense to make this a constant?  `ALL_SENSORS` or something like that? This would reduce the risk of typos and should also be used in the comparisons above.

Reply:
Okay